tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +127,34 @@ describe('run', () => {
     });
   });
 
+  it('passes single url parameter to Firefox when specified', () => {
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+    const testArgs = ['--url', 'www.example.com'];
+
+    return cmd.run({startUrl: 'www.example.com'}).then(() => {
+      assert.equal(firefoxApp.run.called, true);
+      assert.deepEqual(firefoxApp.run.firstCall.args[1].binaryArgs,
+                   testArgs);
```

Comment:
nit, align this line to the first letter of the first function parameter on the previous line.